docs/01-overview/02-main-areas/application-connectivity/ra-02-configuring-runtime.md

```diff
@@ -1,19 +1,15 @@
 ---
 title: Configuring the Runtime
```

Comment:
The document is duplicated. See `docs/05-technical-reference/ra-01-configuring-runtime.md`.